packages/firefuel/test/src/order_by_test.dart

```diff
@@ -5,6 +5,14 @@ import 'package:firefuel/firefuel.dart';
 void main() {
   const String testField = 'testField';
   group('$OrderBy', () {
+    group('#docId', () {
```

Comment:
The hash signifies that you're testing an instance level method where a dot signifies you're testing a class level method (note, both of these are ruby conventions). 

There's no symbol that I know of to say you're testing a named constructor, so we should make one up! Choose a symbol that's not a `#` or a `.` and we'll use that to signify we're testing named constructors.